src/services/puzzlebang/puzzlebang-validators.ts

```diff
@@ -1,7 +1,7 @@
 import { z } from "zod";
 
 export const PuzzlebangCompleteRequestValidator = z.object({
-    // userId: z.string(),
-    email: z.string().email(),
-    puzzleId: z.string(),
+    userId: z.string(),
```

Comment:
I don't think PuzzleBang will know the internal userIds for attendees, unless y'all are moving away from emails..?